src/foam/dao/EasyDAO.js

```diff
@@ -227,6 +303,12 @@ return delegate;
       generateJava: false,
       value: false
     },
+    {
+      /** Enable standard authorization. */
+      class: 'Boolean',
+      name: 'authorize',
+      value: false
```

Comment:
a suggestion, maybe should be a part of second secure refactoring ... Should the default be true?

Reply:
Yes, this is coming later.  This is phase I, we can't convert everything at once.